code/Alert.tsx

```diff
@@ -1,74 +1,75 @@
-import { Alert as AntAlert, Icon } from 'antd'
-import * as React from 'react'
-import { Frame, FrameProps, PropertyControls, ControlType } from 'framer'
-import './App.css'
-import { ReactNode } from 'react'
+import { Alert as AntAlert, Icon } from "antd"
```

Comment:
Sorry for minor formatting differences here...I didn't realise " was used instead of '. The same goes for indentation it looks a bit different here sorry.